api.proto

```diff
@@ -3,13 +3,28 @@ package proto;
 
 option java_package = "io.liftbridge.proto";
 
+// Custom Stream Config
+enum   CompactEnabled {     
+    UNKNOWN  = 0;
```

Comment:
```suggestion
    DEFAULT  = 0;
```

Reply:
I think we should call this `DEFAULT` instead of `UNKNOWN` since it means to take the server-configured default setting.